src/test/docker/travis/Dockerfile

```diff
@@ -65,7 +65,11 @@ RUN mkdir -p /var/run/munge \
  && chmod 600 /etc/munge/munge.key
 
 COPY entrypoint.sh /usr/local/sbin/
+COPY bashrc /tmp
+RUN cat /tmp/bashrc >> ~flux/.bashrc \
```

Comment:
TIL about `~USERNAME`.  That's an awesome expansion trick.  While googling it, I also learned about `~-` which is equivalent to `$OLDPWD`.  :exploding_head: